tools/topology/topology2/platform/intel/sdw-jack-generic.conf

```diff
@@ -0,0 +1,156 @@
+#
+# List of all DAIs
+#
+#ALH Index: 0, Direction: duplex
+Object.Dai.ALH [
+	{
+		dai_index 0
+		id 		$SDW_JACK_OUT_BE_ID
+		direction	"playback"
+		name		$SDW_JACK_OUT_STREAM
+		default_hw_conf_id	0
+		rate			48000
+		channels		2
+
+		Object.Base.hw_config.1 {
+			id	0
+			name	"ALH2"
+		}
+	}
+	{
+		dai_index 10
+		id 		$SDW_JACK_IN_BE_ID
+		direction	"capture"
+		name		$SDW_JACK_IN_STREAM
+		default_hw_conf_id	1
+		rate			48000
+		channels		2
+
+		Object.Base.hw_config.1 {
+			id	1
+			name	"ALH3"
+		}
+	}
+]
+
+#
+# Pipeline definitions
+#
+
+# Pipeline ID:1 PCM ID: 0
```

Comment:
this comment makes no sense. There are 3 pipelines below and no PCMs unlit later

Reply:
> this comment makes no sense. There are 3 pipelines below and no PCMs unlit later

Removed it, too. I have no idea why we need those comments. It was copied from the original csvs-sdw.conf.